Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'tilt', '1.4.1'
 gem 'uglifier'
 gem 'uk_postcode', '~> 1.0.1'
 gem 'unicorn', '4.8.3'
+gem 'hashugar'
```

Comment:
That is useful, thanks @floehopper. I spent a short while looking for the code in gds-api-adapters but gave up when it wasn't immediately obvious.